index.md

```diff
@@ -1,35 +1,20 @@
 ---
 layout: page
-title: Kì Học Bị Thiếu Của Giáo Trình Khoa Học Máy Tính
+title: The Missing Semester - kỳ học bổ sung các kiến thức hữu ích của ngành khoa học máy tính
```

Comment:
Love having the English tittle here, but I think the description afterwards might be slightly long. Perhaps changing to 
"Kỳ Học Bổ Sung Về Khoa Học Máy Tính" would be a good choice.